File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.java
import java.util.*;

class Solution {
    public int solution(int[] A, int[] B) {
        int ans = 0;
        Arrays.sort(A);
        Arrays.sort(B);
        
        for (int i = 0; i < A.length; i++) {
            ans += (A[i] * B[B.length - i - 1]);
        }
        return ans;
    }
}

File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.js
function solution(A,B){
    A.sort((o1, o2) => o1 - o2);
    B.sort((o1, o2) => o2 - o1);
    
    let index = 0;
    return A.reduce((acc, cur) => acc += (cur * B[index++]), 0);
}